### PR TITLE
Fix syntax

### PR DESCRIPTION
**Pull request by rafaelstz**
## Purpose of this pull request

Fixing syntaxes of English and Markdown.

Screenshot 2019-12-01 18 58 36

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/custom_js.html

## Links to Magento source code

- https://github.com/magento/devdocs/blob/master/src/guides/v2.3/javascript-dev-guide/javascript/custom_js.md

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by meker12** [comment]
running tests

File: src/guides/v2.3/javascript-dev-guide/javascript/custom_js.md [hunk 1]
```diff
@@ -160,10 +160,9 @@ $(mage.apply);
...
-Related topic
+Related topics
 
 -  [JavaScript resources in Magento]({{ page.baseurl }}/javascript-dev-guide/javascript/js-resources.html)
 -  [About AMD modules and RequireJS]({{ page.baseurl }}/javascript-dev-guide/javascript/js-resources.html)
-
-[JavaScript initialization]: {{page.baseurl}}/javascript-dev-guide/javascript/js_init.html
-v2.3/performance-best-practices/advanced-js-bundling.html
+-  [JavaScript initialization]({{page.baseurl}}/javascript-dev-guide/javascript/js_init.html)
+-  [Advanced JavaScript bundling]({{page.baseurl}}/v2.3/performance-best-practices/advanced-js-bundling.html)
```
Comment (meker12) [inline src/guides/v2.3/javascript-dev-guide/javascript/custom_js.md]:
```suggestion
-  [Advanced JavaScript bundling]({{page.baseurl}}/performance-best-practices/advanced-js-bundling.html)
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**